nipype/algorithms/filters.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Signal processing tools
+"""
+from __future__ import (print_function, division, unicode_literals,
+                        absolute_import)
+
+from ..utils.filemanip import fname_presuffix
+from ..interfaces.base import (traits, TraitedSpec, SimpleInterface,
+                               BaseInterfaceInputSpec, File)
+
+
+class BandpassInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='functional data')
+    freq_low = traits.Range(0.0, min=0.0, usedefault=True, desc='low frequency cutoff')
```

Comment:
perhaps specify that these are in Hz. (similar to the original desc). and may be we can add the semantics of 0.

freq_hi: the default of 0 sets high pass cutoff to 0
freq_low: the default of 0 sets low pass cutoff to Nyquist